webapp/channels/src/components/post_markdown/system_message_helpers.tsx

```diff
@@ -424,6 +425,13 @@ export function renderSystemMessage(post: Post, currentTeam: Team, channel: Chan
                 messageData={messageData}
             />
         );
+    } else if (post.type === Posts.POST_TYPES.GM_CONVERTED_TO_CHANNEL) {
+        // This is rendered via a separate component instead of registering in
+        // systemMessageRenderers because we need to format a list with keeping i18n support
+        // which cannot be done outside a react component.
```

Comment:
One of these days, we should really make all of the functions in this file into proper React components. They're basically components already except they don't take their arguments in as a props object

Reply:
👍